src/utils/validation.ts

```diff
@@ -42,4 +42,9 @@ function isValidUrl(url: string) {
   return URLRegex.test(url)
 }
 
-export { isValidEmail, isValidUrl }
+export const useValidator = () => {
+  return {
+    email: isValidEmail,
+    url: isValidUrl,
+  }
+}
```

Comment:
Why this change?

Reply:
This is the Updated Validator Hook throughout other codes, and being used in the newer implementation of login.